appengine/flexible/multiple_services/api_gateway/gateway/services_config.py

```diff
@@ -0,0 +1,39 @@
+import os
+from flask import Flask
+
+#to add services insert key value pair of the name of the service and
+#the port you want it to run on when running locally
+SERVICES = {
+    'default': 8000,
+    'static': 8001
+}
+
+def make_app(name):
+    app = Flask(name)
+    environment = 'production' if os.environ.get('GAE_LONG_APP_ID') else 'development'
```

Comment:
For the time being, I recommend using `os.environ.get('GAE_INSTANCE', os.environ.get('GAE_MODULE_INSTANCE'))`.